service/history/task/transfer_active_task_executor.go

```diff
@@ -1700,11 +1695,14 @@ func (t *transferActiveTaskExecutor) processParentClosePolicy(
 
 	for initiatedID, childDomainID := range sameClusterChildDomainIDs {
 		childInfo := childInfos[initiatedID]
-		if err := t.applyParentClosePolicy(
+		if err := applyParentClosePolicy(
```

Comment:
I thought we agreed on sending a signal to the system workflow for processing parent close policy if the child domain may potentially be cross-cluster? Or that will be in another pr? 

I am totally fine with having another pr for it. Actually, you can add a TODO here and I can implement that logic. Let's get this pr landed first.

Reply:
Let me put a comment here, I'm not 100% sure if we need to use a system workflow to apply parent policy.